### ISSUE TITLE
website ssl

### ISSUE THREAD

**Issue by ndwaileebe** [CodeForTucson/codefortucson-site#99]
I believe it is worth noting that the website breaks when loaded in SSL.

**Comment by DainialPadraig** [comment]
Good catch. There's also a security vulnerability in the current codebase with the version of Ruby Jekyll is using.
